packages/components/tables/_tables.scss

```diff
@@ -22,6 +22,16 @@
   }
 }
 
+.nhsuk-table {
+  @include nhsuk-font($size: 19);
```

Comment:
What is the reason for adding this? Could this perhaps mess up any specific type styling that people may have added in their tables?

Reply:
It mirrors [what GOV.UK do](https://github.com/alphagov/govuk-frontend/blob/main/packages/govuk-frontend/src/govuk/components/table/_index.scss#L3) - providing some default sizing. I'd hope it wouldn't override any specific sizing as it should be less specific than any added by users. Not 100% confident about that though.